core/src/main/java/org/apache/druid/indexer/TaskInfo.java

```diff
@@ -35,9 +35,12 @@
   private final String dataSource;
   @Nullable
   private final EntryType task;
+  @Nullable
+  private final String groupId;
```

Comment:
This information is duplicate since it's already in `task`. The caller can get `groupId` by calling `getTask().getGroupId()`.

Reply:
I see, thank you, removed groupId from here.